aws/service_v2.go

```diff
@@ -1021,7 +1094,7 @@ func WAFRegionalClient(ctx context.Context, d *plugin.QueryData) (*wafregional.C
 }
 
 func WAFV2Client(ctx context.Context, d *plugin.QueryData, region string) (*wafv2.Client, error) {
-	validRegions, err := SupportedRegionsForService(ctx, d, wafv2Enpoint.EndpointsID)
+	validRegions, err := GetSupportedRegionsForClient(ctx, d, wafv2Enpoint.EndpointsID)
```

Comment:
Do the tables that use this client already check for `nil` service? I didn't see any changes in this PR for that, but not sure if they were already handling it.